### PR TITLE
[FIX] adapted user parameter parser for idXML files

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by timosachsenberg** [comment]
idXML was already storing all types of user parameters so I updated the schema and the parser to reflect this.

**Comment by timosachsenberg** [comment]
travis error unrelated

**Comment by timosachsenberg** [comment]
any comments?

File: src/openms/source/FORMAT/IdXMLFile.cpp [hunk 1]
```diff
@@ -673,24 +673,35 @@ namespace OpenMS
...
+      { 
+        last_meta_->setMetaValue(name, attributeAsDoubleList_(attributes, "value")); 
+      } 
+      else if (type == "stringList") 
+      { 
+        last_meta_->setMetaValue(name, attributeAsStringList_(attributes, "value")); 
+      } 
+      else 
+      { 
+        fatalError(LOAD, String("Invalid userParam type '") + type + "'");
```
Comment (cbielow) [inline src/openms/source/FORMAT/IdXMLFile.cpp]: what happened to 'name' in the message?! might be useful for debug?

**Comment by cbielow** [comment]
+1

**Pull request merged**